.gitignore

```
@@ -13,4 +13,7 @@ vegetation/profile_output.prof# pixi environments
 logs
 
 *.zarr
-*.zarr.sync
\ No newline at end of file
+*.zarr.sync
+
+.DS_Store
+*/.DS_Store
```

```suggestion
**/.DS_Store
```

Will ignore it recursively!